docs/source/normalization.rst

```diff
@@ -20,15 +20,223 @@ An artifact is any field or subset of data within a record that bears meaning be
 
 ``Artifact Extractor`` Lambda function will build an artifacts inventory based on S3 and Athena services. It enables users to search for all artifacts across whole infrastructure from a single Athena table.
 
+Architecture
+============
+
+.. figure:: ../images/normalization-arch.png
```

Comment:
this image could use some love (like attention to detail with capitalized words vs non-capitalized words, spacing between labels + images, etc)... can you share this with me and I can tweak it?